rmk-config/src/lib.rs

```diff
@@ -346,6 +346,8 @@ pub struct BehaviorConfig {
     pub tap_hold: Option<TapHoldConfig>,
     pub one_shot: Option<OneShotConfig>,
     pub combo: Option<CombosConfig>,
+    #[serde(alias = "macro")]
+    pub macros: Option<MacrosConfig>,
```

Comment:
is there any particular reason for not using `pub macro: ..`?

Reply:
It's reserved: `expected identifier, found reserved keyword`.
it could be defined as `pub r#macro:`, but then we need the `r#` notation everywhere, e.g., `behavior.r#macro`